src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -43,6 +43,7 @@ public function __construct(CurlClientState $multi, $ch, array $options = null,
         $this->multi = $multi;
 
         if (\is_resource($ch)) {
+            unset($multi->handlesActivity[(int) $ch]);
```

Comment:
A little suggestion. Maybe instead of `(int)` let's use the function `get_resource_id` from `symfony/polyfill-php80` ?

Reply:
Thanks for the proposal. I'm going to keep this as is: there's no downside to keeping as is and no upside to doing the change.